common/directoryBlock/directoryBlockHeader.go

```diff
@@ -16,23 +16,28 @@ import (
 
 var _ = fmt.Print
 
+// DBlockHeader contains information related to a specific directory block.
 type DBlockHeader struct {
-	Version   byte   `json:"version"`
-	NetworkID uint32 `json:"networkid"`
+	Version   byte   `json:"version"`   // The nework version, only supported version seems to be constants.VERSION_0
+	NetworkID uint32 `json:"networkid"` // Three supported networks in constants: MAIN_NETWORK_ID, TEST_NETWORK_ID, LOCAL_NETWORK_ID
 
```

Comment:
should remove blank lines to make the json tags easier to read

Reply:
done, check now